ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Synchronize.scala

```diff
@@ -34,13 +29,6 @@ object Synchronize {
       bob <- beta.allocateParty()
       _ <- alpha.waitForParties(Set(beta), Set(alice, bob))
       _ <- beta.waitForParties(Set(alpha), Set(alice, bob))
-      factory <- alpha.create(alice, AgreementFactory(bob, alice))
```

Comment:
@digital-asset/kv-participant Does this simplification make sense to you?

Reply:
@hubert-da good point, I think it makes sense as both participants see the newly allocated parties therefore they should have any contracts created before this call. Let's see what the participant team thinks.

Reply:
I think that's sane as well.